R/livy_connection.R

```diff
@@ -601,7 +610,7 @@ livy_connection <- function(master,
 
   livy_validate_master(master, config)
 
-  if (!is.null(version) && !spark_config_value(config, "sparklyr.livy.sources", FALSE)) {
+  if (!spark_config_value(config, "sparklyr.livy.sources", FALSE)) {
     extensions <- spark_dependencies_from_extensions(version, extensions, config)
 
     config$livy.jars <- as.character(c(livy_connection_jars(config, version), extensions$catalog_jars))
```

Comment:
If getting the jars from github fails for whatever reason, can we fall back to uploading from local (since I presume they're available in `inst/java`)?

Reply:
We are removing them completely, there is no fallback anymore with this PR! +1 -1?

Reply:
Oh you mean uploading the jars... unfortunately, that's not supported by Livy :( They need to be downloaded from somewhere or manually copied into the Livy server. Would be nice!